service/http.directory/src/test/scripts/org/kaazing/gateway/service/http/directory/continuous.symlinks.out.of.directory.rpt

```diff
@@ -0,0 +1,7 @@
+connect http://localhost:8701/linkToOutsideAllowed
```

Comment:
its not clear from this test, but that this is following a link to a directory or a file.  Maybe rename the symlink